operator/pkg/awsprovider/launchtemplate/reconciler.go

```diff
@@ -212,6 +208,7 @@ var (
 #!/bin/bash
 yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
 /etc/eks/bootstrap.sh %s.%s \
+    --dns-cluster-ip %s \
```

Comment:
could you add the issue here in comments as todo ?

Reply:
I have added it where the const is defined